plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -120,9 +120,10 @@
     description:
       - Target drive's backing file's data format.
       - Used only with clone
+      - Use format: null and full: no for linked clone
     type: str
     default: qcow2
-    choices: [ "cloop", "cow", "qcow", "qcow2", "qed", "raw", "vmdk" ]
+    choices: [ "cloop", "cow", "qcow", "qcow2", "qed", "raw", "vmdk", null ]
```

Comment:
```suggestion
    choices: [ "cloop", "cow", "qcow", "qcow2", "qed", "raw", "vmdk", null ]
```
Please use a string, not another type.

Reply:
Can I use an empty string?

Reply:
How about `unspecified`? An empty string is not really good UX, since it is easy to miss and hard to enter (if you don't add quotes, you end up with `null` instead).